src/main/java/core/basesyntax/figures/Figure.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.AreaCalculator;
+import core.basesyntax.Drawable;
+
+public abstract class Figure implements AreaCalculator, Drawable {
+    private String color;

```

Comment:
In my opinion, getColor should remain a String. Because I call this method in every shape. Why add .name().toLowerCase() everywhere when I can do it in one place.

Reply:
The best practice is to store a value in model classes in base type (enum). This suggestion make code more consistent and decrease a possible mistakes due to converting values when the program will grow. 
It is not a problem to use ` .name().toLowerCase()` in this task, but if you want to improve the solution you can upgrade the enum Color class with constructor and new field `name` that will save lowercase name value.
https://www.programiz.com/java-programming/enum-constructor

Reply:
> 
https://github.com/mate-academy/jv-oop-advanced/pull/1048#discussion_r1176037226

I had this implementation, but I was asked to fix it